templates/index.html

```diff
@@ -263,25 +279,21 @@ <h4 class="pricing__option-cost">&dollar;10
 	</section>
 
 	<!-- News -->
-	<section class="row">
-		<div class="inner-wrapper">
-			<div class="twelve-col">
-				<h2>The latest MAAS news</h2>
-			</div>
-			<div id="insights-cloud-feed">
-				<div class="equal-height divider--vertical twelve-col">
-					{% get_rss_feed "https://insights.ubuntu.com/tag/maas/feed" limit=3 as insights_feed %}
-					{% for item in insights_feed %}
-						<article class="four-col divider__col {% if forloop.last %}last-col{% endif %}">
-							<h3><a href="{{ item.link }}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : 'maas.io homepage news link', 'eventLabel' : '{{ item.title|escapejs }}', 'eventValue' : '{{ item.link }}' });">{{ item.title }}</a></h3>
-							<footer>
-								<time pubdate datetime="{{ item.updated }}">{{ item.updated_datetime|date:"j F Y" }}</time>
-							</footer>
-						</article>
-					{% endfor %}
-				</div>
+	<section class="p-strip--light is-deep">
+		<div class="row">
+			<h2>The latest MAAS news</h2>
+			<div id="insights-cloud-feed" class="p-divider col-12">
+				{# get_rss_feed "https://insights.ubuntu.com/tag/maas/feed" limit=3 as insights_feed #}
+				{# for item in insights_feed #}
+					<article class="col-4 p-divider__block">
+						<h3 class="p-heading--four"><a class="p-link--strong" href="{{ item.link }}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'insights', 'eventAction' : 'maas.io homepage news link', 'eventLabel' : '{{ item.title|escapejs }}', 'eventValue' : '{{ item.link }}' });">{{ item.title }}</a></h3>
```

Comment:
Also the feed is not loading any more? 

![image](https://user-images.githubusercontent.com/1201618/27487940-2716ecf8-582d-11e7-9ae8-45601886309f.png)